en/power_module/sky-drones_smartap-pdb.md

```diff
@@ -0,0 +1,72 @@
+# Sky-Drones SmartAP PDB
+
+[SmartAP PDB](https://sky-drones.com/power/smartap-pdb.html) (Power Distribution Board) is used to simplify the distribution of power from one or more batteries to ESCs (motors), flight controller, and other peripherals.
+It also acts as a [power module](../power_module/README.md), enabling battery voltage and current measurements.
+SmartAP PDB makes connecting high-power lines easier and much more reliable.
+
+![SmartAP PDB](../../assets/hardware/power_module/sky-drones_smartap-pdb/smartap-pdb-top-side.jpg)
+
+## Specifications
+
+- Size: 65x65 mm, 4x M3 mounting holes
+- Input voltage up to 60 Volts (14S)
+- Capability to handle extremenly high currents (peak current up to 400A)
+- Power input from the main battery, possibility to connect up to 2 independent batteries
+- 12 pairs of pads (6 on top, 6 on bottom) for powering up to 12 motors
+- Integrated voltage and current sensors with L/C filters
+- Precise Hall Effect based current measurements
+- Integrated DC-DC converter from 10-60 V input (up to 14S battery) to 5V / 5A output to power peripherals
+- Integrated DC-DC converter from 10-60 V input (up to 14S battery) to 12V / 5A output to power peripherals
+- 5V and 12V power output terminals (standard 2.54mm/0.1" connectors)
+- Integrated electromagnetic sounder (buzzer)
+- Power output for the flight controller (both 5V regulated and battery voltage level output)
+
+
+## Size and Weight
+
+- Length: 65mm
+- Width: 65mm
+- Height: 14mm
+- Weight: 16g
+
+## PX4 Configuration
+
+[Battery and Power Module Setup](../config/battery.md) describes how to configure the battery setting for a power module.
```

Comment:
This link is the key for me - without this, the configuration settings are not helpful to a beginner.

The settings I see on the linked screen are there: 
![image](https://user-images.githubusercontent.com/5368500/204720199-757a01cb-d7d5-41d2-a864-79854ad9faf3.png)

Can we update the values below to have matching text? E.g. "Amps per volt" - unless you have noted that QGC has changed and the documentation in this image is wrong (in which case job for me to update!)

Reply:
Thanks! Updated to match the text. I think let's leave it as per the image above for now and once it's updated on the dedicated battery settings page I'll update it here too.